x-pack/plugins/maps/public/embeddable/types.ts

```diff
@@ -27,7 +32,7 @@ export interface MapEmbeddableConfig {
 
 export interface MapEmbeddableInput extends EmbeddableInput {
   timeRange?: TimeRange;
-  filters: Filter[];
+  filters: FieldFilter[];
```

Comment:
Why was this typing change required?

Reply:
It wasn't allowing adding exists filter.